goss-testing/automated/run-ncn-tests.sh

```diff
@@ -83,6 +83,15 @@ function sw_admin_pw_set {
     return 0
 }
 
+function SSHPASS {
```

Comment:
If the user is already going to be setting the SW_ADMIN_PASSWORD variable, why not just check for that, and export it as SSHPASS inside your script? That way they don't need to enter the same password twice.

Reply:
Also, I don't see this function being called from anywhere.